rest-service/manager_rest/rest/resources_v2_1/deployment_update.py

```diff
@@ -244,3 +240,16 @@ def get(self,
                 substr_filters=search
             )
         return deployment_updates
+
+
+def _get_blueprint_abs_path(server_root, blueprint):
```

Comment:
very tied to plugins update, should be a more relevant name

Reply:
Changed to `_check_if_temp_of_plugins_update_and_get_abs_path`, is it better?

Reply:
_get_plugin_update_blueprint_abs_path